examplesRSECTION/high_level_functions_demos/basic_objects_demo.js

```diff
@@ -116,6 +115,30 @@ stressPoint.OnLine(undefined, 6, [0.75, undefined]);
 stressPoint.OnLine(undefined, 6, [undefined, 0.01, false]);     // absolute distances
 stressPoint.OnElement(undefined, 5, [0.10, undefined]);   // relative distances
 
+/*********************************************** Bar ********************************************/
```

Comment:
make separete demo for concrete stuff

Reply:
Created (concrete_reinforcement_demo.js)